instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld

```diff
@@ -15,8 +15,8 @@
   "copyright": null,
   "custodian": null,
   "description": null,
-  "digitalIdentifier": null,
-  "fullDocumentation": null,
+  "digitalIdentifier": "https://scicrunch.org/resolver/RRID:SCR_017124",

```

Comment:
this should be:
`"digitalIdentifier": {"@id": "https://scicrunch.org/resolver/RRID:SCR_017124"},`

Reply:
Fixed.